File: src/main/java/rain/gtetcore/client/ClientProxy.java
package rain.gtetcore.client;

import net.minecraftforge.api.distmarker.Dist;
import net.minecraftforge.api.distmarker.OnlyIn;
import net.minecraftforge.eventbus.api.IEventBus;
import net.minecraftforge.fml.javafmlmod.FMLJavaModLoadingContext;
import rain.gtetcore.common.CommonProxy;


@OnlyIn(Dist.CLIENT)
public class ClientProxy extends CommonProxy {
	public ClientProxy() {
		super();
		@SuppressWarnings ("Deprecated")
		IEventBus eventBus =FMLJavaModLoadingContext.get ().getModEventBus ();

	}
}

File: src/main/kotlin/rain/gtetcore/gtet/client/ClientProxy.kt
package rain.gtetcore.gtet.client

import net.minecraftforge.api.distmarker.Dist
import net.minecraftforge.api.distmarker.OnlyIn
import net.minecraftforge.eventbus.api.IEventBus
import net.minecraftforge.fml.javafmlmod.FMLJavaModLoadingContext
import rain.gtetcore.gtet.common.CommonProxy

/**
 * 放只有客户端需要加载的类
 * */
@OnlyIn(Dist.CLIENT)
open class ClientProxy() : CommonProxy() {
    init {
        @SuppressWarnings("deprecated")//byd这玩意为什么总要提醒，我byd是1.20.1啊，F**k
        val bus: IEventBus = FMLJavaModLoadingContext.get().modEventBus
    }
}
